topics/firefox/index.md

```diff
@@ -0,0 +1,13 @@
+---
+aliases: seamonkey, waterfox, palemoon, icecat, iceweasel, torbrowser
+created_by: Mozilla
+display_name: Mozilla Firefox
+github_url: https://github.com/mozilla
+released: September 23, 2002
+short_description: An advanced, free, open-source web browser.
+topic: firefox
+url: https://mozilla.org/firefox
+wikipedia_url: https://en.wikipedia.org/wiki/Firefox
+related: WebExtension
```

Comment:
https://github.com/topics/WebExtension 404s, but https://github.com/topics/web-extension works. Can you change this to `web-extension`?

Reply:
I guess that ```web-extension``` should be an alias to ```WebExtension``` since `WebExtension` is an official name.